android-agent/src/main/java/io/opentelemetry/android/initialization/InitializationEvents.kt

```diff
@@ -25,12 +27,39 @@ interface InitializationEvents {
     fun spanExporterInitialized(spanExporter: SpanExporter)
 
     companion object {
-        @JvmField
+        private var instance: InitializationEvents? = null
+
+        @JvmStatic
+        fun get(): InitializationEvents {
+            if (instance == null) {
+                val initializationEvents = load(InitializationEvents::class.java).firstOrNull()
+                if (initializationEvents != null) {
+                    set(initializationEvents)
+                } else {
+                    set(NO_OP)
+                }
+            }
+
+            return instance!!
+        }
+
+        @JvmStatic
+        fun set(initializationEvents: InitializationEvents) {
```

Comment:
Is this just visible for testing? Could use a comment clarifying such.

Reply:
Or can it be made internal? It would be an error for a user to call this multiple times etc.

Reply:
Good point, I don't think there's any reason to expose it for now, so I've moved it to the internal package.